### ISSUE TITLE
User stats for EDS

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#100]
Derrick mentioned a need for user stats on the EDS itself.  I realize that this will reveal stats for users pinging on the EDS outside of IOOS (i.e. coast guard, etc).  Regardless, these are metrics we could at least talk about with the caveats.  Is there a way to provide this?

**Comment by Bobfrat** [comment]
Yes this should be possible. I don't think we'd be able to leverage Google Analytics but we should be able to log requests made against the IOOS EDS key and create monthly reports. Do you know what kind of metrics we'd like to report?

**Comment by kbailey-noaa** [comment]
I think something like this would be useful:
http://dods.ndbc.noaa.gov/stats/oceansites/
The monthly reports show more granularity. 
There are 2 goals: 
1) To understand, in broad terms, who is accessing the RA model output from the EDS server in general - whether it's through a client other than the IOOS EDS Model Viewer or not. Who are they (geographic region, industry sector, business if possible...), how often, what layers are they grabbing.  It would be great to tell a story of how RA models are popular among an EDS audience for a variety of purposes (e.g. oil spill response, etc).
2) To understand who is accessing the THREDDS via the IOOS EDS Model Viewer - how often, who are they, what layers are they grabbing.  This would indicate IOOS Model Viewer / Oceansmap THREDDS users.

**Comment by kknee** [comment]
Kathy -so a monthly CSV of all models related to the IOOS EDS key?  Or the the USCG key(s), or all users of EDS combined?

**Comment by kbailey-noaa** [comment]
Both the IOOS EDS key as well as the USCG key(s).  I think it would be good to differentiate since it gives some insight into how the data are used and/or who the users are.  Combining would lose that insight.

**Comment by kknee** [comment]
@kbailey-noaa the USCG key CSV was emailed out today - that summary detailed data requests.  Is that also what you want for the EDS key?  Or WMS requests?

**Comment by kknee** [comment]
@kbailey-noaa monthly report for USCG key was sent and that process is automated.  The EDS key doesn't haven any data requests, just WMS requests so we will have to do something differently there (assuming you want WMS requests).

**Comment by kbailey-noaa** [comment]
Let's table this and focus on GA for now.  I think we do want both WMS and data requests but I'm not sure it's worth the effort at this point if there aren't really any data requests to track.

**Comment by kbailey-noaa** [comment]
@kknee Can you clarify the statement that the EDS key doesn't have any data requests? I'm not sure what that means.  Do you mean no users are accessing the eds.ioos.us/thredds access options? (e.g. https://eds.ioos.us/thredds/catalog/EDS/NOS_GOMOFS/catalog.html?dataset=EDS/NOS_GOMOFS/nos_gomofs_forecast_20181121.nc).
